snapcraft/plugins/v2/python.py

```diff
@@ -119,15 +119,7 @@ def get_build_commands(self) -> List[str]:
         # $SNAPCRAFT_PYTHON_INTERPRETER (as -m venv will create the link to the
         # interpreter invoked).
         build_commands = [
-            # Determine python based on PATH.
-            'SNAPCRAFT_PYTHON_PATH="${SNAPCRAFT_PYTHON_PATH:-$(which "${SNAPCRAFT_PYTHON_INTERPRETER}")}"',
-            # Resolve python, in case it is a link, e.g.:
-            # (1) /home/ubuntu/.venv/snapcraft/bin/python3 -> /usr/bin/python3.8
-            # (2) /usr/bin/python3 -> /usr/bin/python3.8
-            # (3) /root/stage/python3 -> /root/stage/python3.8
-            # (4) /root/parts/<part>/install/usr/bin/python3 -> /root/parts/<part>/install/usr/bin/python3.8
-            'SNAPCRAFT_PYTHON_PATH="$(readlink -e "${SNAPCRAFT_PYTHON_PATH}")"',
-            '"${SNAPCRAFT_PYTHON_PATH}" -m venv ${SNAPCRAFT_PYTHON_VENV_ARGS} "${SNAPCRAFT_PART_INSTALL}"',
+            '"${SNAPCRAFT_PYTHON_INTERPRETER}" -m venv ${SNAPCRAFT_PYTHON_VENV_ARGS} "${SNAPCRAFT_PART_INSTALL}"',
```

Comment:
It could be useful for later reference to explain in a comment why we need the separate venv for each part.